be/test/formats/parquet/encoding_test.cpp

```diff
@@ -194,98 +201,101 @@ struct DecoderChecker<Slice, is_dictionary> {
-        ASSERT_TRUE(st.ok());
+                decoder->set_data(encoded_data);
+                auto st = decoder->next_batch(values.size(), ColumnContentType::VALUE, column.get());
+                ASSERT_TRUE(st.ok());
 
-        const auto* check = (const Slice*)column->raw_data();
-        for (auto value : values) {
-            ASSERT_EQ(value, *check);
-            check++;
-        }
+                const auto* check = (const Slice*)column->raw_data();
+                for (auto value : values) {
+                    ASSERT_EQ(value, *check);
+                    check++;
+                }
 
-        if (!is_dictionary) {
-            // out-of-bounds access
-            st = decoder->next_batch(values.size(), ColumnContentType::VALUE, column.get());
-            ASSERT_FALSE(st.ok());
-        }
-    }
-    // skip+read
-    {
-        size_t values_to_skip = values.size() / 2;
-        size_t remain_values = values.size() - values_to_skip;
+                if (!is_dictionary) {
+                    // out-of-bounds access
+                    st = decoder->next_batch(values.size(), ColumnContentType::VALUE, column.get());
+                    ASSERT_FALSE(st.ok());
+                }
+            }
+            // skip+read
+            {
+                size_t values_to_skip = values.size() / 2;
+                size_t remain_values = values.size() - values_to_skip;
 
-        auto column = starrocks::BinaryColumn::create();
+                auto column = starrocks::BinaryColumn::create();
 
-        decoder->set_data(encoded_data);
-        auto st = decoder->skip(values_to_skip);
-        st = decoder->next_batch(remain_values, ColumnContentType::VALUE, column.get());
-        ASSERT_TRUE(st.ok());
+                decoder->set_data(encoded_data);
+                auto st = decoder->skip(values_to_skip);
+                st = decoder->next_batch(remain_values, ColumnContentType::VALUE, column.get());
+                ASSERT_TRUE(st.ok());
 
-        const auto* check = (const Slice*)column->raw_data();
-        for (size_t i = 0; i < remain_values; i++) {
-            EXPECT_EQ(values[values_to_skip + i], check[i]);
-        }
+                const auto* check = (const Slice*)column->raw_data();
+                for (size_t i = 0; i < remain_values; i++) {
+                    EXPECT_EQ(values[values_to_skip + i], check[i]);
+                }
 
-        if (!is_dictionary) {
-            // out-of-bounds access
-            st = decoder->skip(2);
-            ASSERT_FALSE(st.ok());
+                if (!is_dictionary) {
+                    // out-of-bounds access
+                    st = decoder->skip(2);
+                    ASSERT_FALSE(st.ok());
+                }
+            }
         }
-    }
-} {
-    // read
-    {
-        auto data_column = starrocks::BinaryColumn::create();
-        auto column = NullableColumn::create(std::move(data_column), NullColumn::create());
+        {
```

Comment:
```
{
    {
        // code
    }
    {
        // code
    }
}
```

should be simplified to 
```
    {
        // code
    }
    {
        // code
    }
```

Avoid confusing to the format tool recognise the `{{ // code }{ // ...}}` which looks like an in-place initialization.

Reply:
thanks.